portal/README.md

```diff
@@ -1,30 +1,74 @@
 # Portal
 
+# Prerequisite
+
+1. Need to start `db` and `redis` container
+2. Need to start authgear servers
```

Comment:
Maybe just tell the reader to follow `../README.md` to get Authgear running first.